_posts/2019-10-08-system-banner.md

```diff
@@ -0,0 +1,26 @@
+---
+permalink: /system-banner.html
+layout: post
+title: Search.gov Login Terms
+tags: status
+description: This page provides access to the Search.gov Application.
```

Comment:
@dawnpm , do we want this page to be indexed & searchable? If not, let's add these tags:
```
sitemap: false
metatags: noindex
```

Reply:
Good call, putting those in.